_posts/2019-09-30-team-working-agreements.md

```diff
@@ -0,0 +1,261 @@
+      not push a working agreement brainstorm so strongly in one direction again.
+    </p>
+  </div>
+</aside>
+
+(ADAM: Add an Adam says... section to share the habits that the Galleries team agreed upon)
+
+Remember that the working agreement is a living document. If the team learns that it missed something in the
+brainstorm, don't hesitate to update the agreement.
+
+### Working agreements are hard to define for teams without a well-established identity.
+
+For a team that has a strong and cohesive identity, the agreement is likely to affirm many things the team is
+already believing and doing. For a less cohesive team the agreement is harder to pin down. Many opinions will
+surface, and they may be in conflict with each other.
+
+While this is indeed challenging, it is also important to note that teams that lack identity value _greatly_ from
+the team agreement brainstorm. Provided there is
+[psychological safety](https://github.com/artsy/README/blob/master/culture/engineering-principles.md#psychological-safety),
+this is a really great time for the team to learn about what matters to each other, and move toward establishing an
+identity.
+
+### Facilitating the brainstorm can be difficult on a distributed team.
+
+Some of our teams were able to brainstorm with everyone on-site. They were able to rely on post-it notes and
+in-person conversations, and read each other's non-verbal communication. Other teams needed to accommodate remote
+members, and used tools like [Miro](http://miro.com/) to facilitate. This _can_ present a challenge...but if you're
+already a distributed team, they are likely the same types of challenges you've worked to overcome for _all_
+meetings.
+
+The differences are worth noting, though. The importance of reading each other's non-verbal communication is
+magnified when you're having conversations about identity. It's quicker to organize and re-organize real-life
+post-its than virtual post-its in an online tool. These are all opportunities to get better at being a distributed
+team.
+
+### It takes effort to make sure one person isn't introducing their bias into the working agreement.
+
+It's important for the team agreement brainstorm to provide an environment in which every team member is heard.
+Even if your team is successful at this, there are still opportunities for individual bias to affect the working
+agreement.
+
+We saw earlier how bias can appear when the examples provided for the brainstorm are too narrowly scoped.
+
+We recognized another bias vector in the distillation of the brainstorm into an actual agreement. If this is
+handled by a single person, it's very possible for them to produce a working agreement that misrepresents _the
+rest_ of the team. To counter this, we've had multiple team members pair on the distillation process. Feedback on
+the initial draft of the agreement is also important to ensure it does not misrepresent the team's ideas.
```

Comment:
I wonder if there are objective ways to avoid this, such as voting on brainstorm items in order to prioritize the most _truly_ popular concerns.